syntax = "proto3";

package datafile;

message DataFile {
  repeated DataHeader header = 1;
  repeated DataRow row = 2;
}

message DataHeader {
  repeated string column_name = 1;
}

message DataRow {
  repeated int64 column_value = 1;
}
